kubeadm-one

```diff
@@ -175,7 +175,7 @@ function main() {
 
     # Initialize the single node kubernetes cluster
     if ! kubectl cluster-info &>/dev/null; then
-        initialize_kubeadm $APISERVER_CERT_EXTRA_SANS $POD_NETWORK_CIDR
+        initialize_kubeadm "$APISERVER_CERT_EXTRA_SANS" "$POD_NETWORK_CIDR"
```

Comment:
It appears that some bash indents are 2 spaces and some are 4. May: make indentation spaces consistent.

Reply:
Will do in next PR.